core/client/fs/src/test/java/alluxio/client/file/cache/LocalCacheManagerTest.java

```diff
@@ -54,66 +51,148 @@
 
   private LocalCacheManager mCacheManager;
   private InstancedConfiguration mConf = ConfigurationTestUtils.defaults();
-  private HashSetMetaStore mMetaStore;
-  private HashMapPageStore mPageStore;
+  private MetaStore mMetaStore;
+  private PageStore mPageStore;
   private CacheEvictor mEvictor;
 
+  @Rule
+  public TemporaryFolder mTemp = new TemporaryFolder();
+
   @Rule
   public final ExpectedException mThrown = ExpectedException.none();
 
   @Before
   public void before() {
     mConf.set(PropertyKey.USER_CLIENT_CACHE_PAGE_SIZE, PAGE_SIZE_BYTES);
     mConf.set(PropertyKey.USER_CLIENT_CACHE_SIZE, CACHE_SIZE_BYTES);
-    mMetaStore = new HashSetMetaStore();
-    mPageStore = new HashMapPageStore();
+    mConf.set(PropertyKey.USER_CLIENT_CACHE_DIR, mTemp.getRoot().getAbsolutePath());
+    mMetaStore = MetaStore.create();
+    mPageStore = PageStore.create(mConf);
```

Comment:
Let's test with real implementation of MetaStore and PageStore instead the mock. FYI